### ISSUE TITLE
ImportError: Module 'curses' is not installed

### ISSUE THREAD

**Issue by blais** [beancount/beancount#51]
**[Original report](https://bitbucket.org/blais/beancount/issue/51) by Saša Janiška (Bitbucket: [gour](https://bitbucket.org/gour), GitHub: [gour](https://github.com/gour)).**

----------------------------------------

While working on tutorial, I've encountered the following:

```
#!python

gour@atmarama ~/financije> bean-report example.beanaccount journal -a Assets:US:BofA:Checking
Traceback (most recent call last):
  File "/usr/bin/bean-report", line 2, in <module>
    from beancount.scripts.report import main; main()
  File "/usr/lib64/python3.4/site-packages/beancount/scripts/report.py", line 225, in main
    chosen_report.render(entries, errors, options_map, args.format, outfile)
  File "/usr/lib64/python3.4/site-packages/beancount/reports/report.py", line 114, in render
    result = render_method(entries, errors, options_map, outfile)
  File "/usr/lib64/python3.4/site-packages/beancount/reports/report.py", line 219, in forward_method
    return fwdfunc(self, real_root, options_map, file)
  File "/usr/lib64/python3.4/site-packages/beancount/reports/journal_reports.py", line 81, in render_real_text
    self._render_text_formats(real_root, options_map, file, journal_text.FORMAT_TEXT)
  File "/usr/lib64/python3.4/site-packages/beancount/reports/journal_reports.py", line 68, in _render_text_formats
    width = self.args.width or misc_utils.get_screen_width()
  File "/usr/lib64/python3.4/site-packages/beancount/utils/misc_utils.py", line 303, in get_screen_width
    import curses
  File "/usr/lib64/python3.4/_import_failed/curses.py", line 17, in <module>
    to install it.""".format(__name__, package))
ImportError: Module 'curses' is not installed.
Use:
  sudo zypper install python3-curses
to install it.

```

Don't know if it's openSUSE or general issue, but isntalling python3-curses does help.

**Comment by blais** [comment]
**Original comment by Saša Janiška (Bitbucket: [gour](https://bitbucket.org/gour), GitHub: [gour](https://github.com/gour)).**

----------------------------------------

Here is the output from bean-doctor:


```
#!bash

gour@atmarama ~/financije> bean-doctor checkdeps
Dependencies:
  python3         : 3.4.3 
  dateutil        : 2.4.2 
  bottle          : 0.12.7 
  ply             : 3.6 
  wget            : 1.16.3 

```

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

curses comes standard with a normal Python installation.
This looks like it's an exotic SUSE problem, SUSE should install this by default.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

I'll add a check in bean-doctor checkdeps for it just in case.

**Comment by blais** [comment]
**Original comment by Saša Janiška (Bitbucket: [gour](https://bitbucket.org/gour), GitHub: [gour](https://github.com/gour)).**

----------------------------------------

> This looks like it's an exotic SUSE problem, SUSE should install this by default.

Thank you. I'm experiencing some other python3-related issues with SUSE and will migrate very soon...just need enough time to do it.

**Comment by blais** [comment]
**Original comment by Martin Blais (Bitbucket: [blais](https://bitbucket.org/blais), GitHub: [blais](https://github.com/blais)).**

----------------------------------------

Fixed #51: Added a version check for 'curses' in 'bean-doctor checkdeps' command.
